nmpolicy/internal/resolver/path.go

```diff
@@ -57,6 +57,10 @@ func (v pathVisitor) visitInterface(inputState interface{}) (interface{}, error)
 }
 
 func (v pathVisitor) visitSlice(originalSlice []interface{}) (interface{}, error) {
+	return v.visitSliceWithoutIndex(originalSlice)
```

Comment:
At next commits it will do more things https://github.com/nmstate/nmpolicy/pull/95/files#diff-54f62b91fe5d46a180c2286f8d68e4bab06e58939878ccbb5e8fbd35e47bade2R62-R65, this is here for completness.